### PR TITLE
Run code formatter on lib/elixir/lib/stream.ex

### PR DESCRIPTION
**Pull request by udaysinghcode**
bin/elixirc lib/elixir/lib/stream.ex -o lib/elixir/ebin
bin/elixir lib/elixir/test/elixir/stream_test.exs
make test

After running all three, this is build passing.

### CODE REVIEW

File: lib/elixir/lib/stream.ex [hunk 1]
```diff
@@ -1284,27 +1428,43 @@ defmodule Stream do
...
       {list, next_acc} when is_list(list) ->
-        do_list_resource(next_acc, next_fun, {:cont, acc}, fun, after_fun,
-                         &Enumerable.List.reduce(list, &1, fun))
+        do_list_resource(
+          next_acc,
+          next_fun,
+          {:cont, acc},
+          fun,
+          after_fun,
+          &Enumerable.List.reduce(list, &1, fun)
```
Comment (josevalim) [inline lib/elixir/lib/stream.ex]: Please move this to a variable so we can keep the call as a single line.

File: lib/elixir/lib/stream.ex [hunk 2]
```diff
@@ -1284,27 +1428,43 @@ defmodule Stream do
...
-        do_enum_resource(next_acc, next_fun, {:cont, acc}, fun, after_fun,
-                         &Enumerable.reduce(enum, &1, inner))
+
+        do_enum_resource(
+          next_acc,
+          next_fun,
+          {:cont, acc},
+          fun,
+          after_fun,
+          &Enumerable.reduce(enum, &1, inner)
```
Comment (josevalim) [inline lib/elixir/lib/stream.ex]: Same here.

File: lib/elixir/lib/stream.ex [hunk 3]
```diff
@@ -1449,28 +1612,37 @@ defimpl Enumerable, for: Stream do
 
   defp do_reduce(%Stream{enum: enum, funs: funs, accs: accs, done: done}, acc, fun) do
     composed = :lists.foldl(fn fun, acc -> fun.(acc) end, fun, funs)
-    do_each(&Enumerable.reduce(enum, &1, composed),
-            done && {done, fun}, :lists.reverse(accs), acc)
+
+    do_each(
+      &Enumerable.reduce(enum, &1, composed),
```
Comment (josevalim) [inline lib/elixir/lib/stream.ex]: Same here.

**Comment by udaysinghcode** [comment]
Alright, perfect, give me a few minutes.

**Comment by udaysinghcode** [comment]
I made a small mistake here, apologies. Give me a few minutes.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Pull request merged**